packages/mobile/src/verify/saga.ts

```diff
@@ -689,6 +688,4 @@ export function* verifySaga() {
   yield takeEvery(fail.type, failSaga)
   yield takeEvery(reset.type, resetSaga)
   yield takeEvery(stop.type, stopSaga)
-  // TODO: this can be calculated in reducer, once we stop using identify/reducer for verification
-  yield takeEvery(Actions.COMPLETE_ATTESTATION_CODE, fetchOnChainDataSaga)
```

Comment:
I śaw your comment about this, are we sure there's not side effect to removing this?